main/ertp/guide/PursesAndPayments.md

```diff
@@ -0,0 +1,317 @@
+# Purses and Payments
+ 
+Digital assets can be any of:
+- Currency-like, such as our imaginary Quatloos currency.
+- Goods-like digital assets, such as magic weapons for use in a game or theater tickets.
+- Other kinds of rights, such as the right to participate in a particular contract.
+
+In ERTP, digital assets exist in either a `purse` or a `payment`:
+- **[`purse`](https://agoric.com/documentation/glossary/#purse)**: Holds
+  a quantity of same-branded digital assets until part or
+  all of them are withdrawn into a `payment`. A new `purse` is created
+  by an `issuer` and can only hold assets of that `issuer`'s `brand`. 
+- **[`payment`](https://agoric.com/documentation/glossary/#payment)**:
+  Holds a quantity of same-branded digital assets to transfer to another party. 
+  A new `payment` is created either with new assets by a `mint` or by 
+  withdrawing assets from a `purse`. It can only hold assets of the same `brand` as
+  that `mint` or `purse`. .
+
+For any `brand`, any number of `purses` or `payments` can hold assets
+of that `brand`. Neither a `purse` nor a `payment` can ever change their
+associated `brand`.
+
+Each non-empty `purse` and `payment` object contains exactly one
+quantity of its digital asset. 
```

Comment:
I think this is a very confusing statement. What does one quantity even mean?  I think we should replace this statement with:

```suggestion
Each `purse` and `payment` object contains a specific amount of digital assets, which may be none at all (`empty` in amountMath terms).
```